data/json/items/ammo.json

```diff
@@ -1260,5 +1260,74 @@
     "color": "white",
     "flags": [ "TRADER_AVOID" ],
     "ammo_type": "glowstick_juice"
+  },
+  {
+    "type": "AMMO",
+    "id": "graphite",
+    "name": { "str_sp": "graphite" },
+    "symbol": "!",
+    "description": "Graphite powder mixed with a clay binder. Produces grey or black marks that are easily erased, but otherwise resistant to moisture, most chemicals, ultraviolet radiation and natural aging.",
```

Comment:
```suggestion
    "description": "Graphite powder mixed with a clay binder.  Produces grey or black marks that are easily erased, but otherwise resistant to moisture, most chemicals, ultraviolet radiation and natural aging.",
```